snap/snapcraft.yaml

```diff
@@ -359,6 +436,56 @@ parts:
     source: snap/local/build-helpers
     stage-packages:
       - curl
+
+  # snapcraft preload is necessary to make postgres just use a different
+  # lockfile location in /dev/shm
+  # NOTE: if this ever breaks in really scary ways when compiling with lots of
+  # warnings, see the comment on the kong part, tldr probably some other C/C++
+  # part in the build broke snapcraft-preload by running before
+  # snapcraft-preload
+  snapcraft-preload:
+    source: https://github.com/sergiusens/snapcraft-preload.git
+    plugin: cmake
+    build-packages:
+      - to arm64:
+          - g++-multilib-arm-linux-gnueabihf
+          - gcc-multilib-arm-linux-gnueabihf
+      - else:
+          - gcc-multilib
+          - g++-multilib
+    stage-packages:
+      - to amd64:
+          - lib32stdc++6
+  gosu:
+    source: https://github.com/tianon/gosu.git
```

Comment:
Can we pin this to the latest release (1.11)?

Reply:
done